### PR TITLE
=htt #19792 Increase file upload directive test timeouts

### PR DESCRIPTION
**Pull request by johanandren**
1s => 3s

Refs #19792

### CODE REVIEW

File: akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileUploadDirectivesSpec.scala [hunk 1]
```diff
@@ -4,13 +4,19 @@
...
+
 import akka.http.scaladsl.model._
 import akka.http.scaladsl.server.{ MissingFormFieldRejection, RoutingSpec }
+import akka.http.scaladsl.testkit.RouteTestTimeout
 import akka.util.ByteString
+import scala.concurrent.duration._
 
 class FileUploadDirectivesSpec extends RoutingSpec {
 
+  // tests touches filesystem, so reqs may take longer than the default of 1.second to complete
```
Comment (ktoso) [inline akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileUploadDirectivesSpec.scala]: Could go all fancy and jim-fs and not touch... (-:
But anyway, LGTM

**Comment by ktoso** [comment]
LGTM, 3 sec likely good for IO

File: akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileUploadDirectivesSpec.scala (hunk 1 above)
Comment (patriknw) [inline akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileUploadDirectivesSpec.scala]: and 1 second is to short in general also

**Comment by patriknw** [comment]
LGTM

**Pull request merged**